### ISSUE TITLE
Generation stuck

### ISSUE THREAD

**Issue by xunkar** [Acly/krita-ai-diffusion#432]
Completed installation using local server, default settings (except installation directory). When opening comfy at localhost IP, generation works fine and fast, even with high resolution pictures. But when running in krita, generation is stuck very early on in the process. Server log says

2024-02-26 20:05:46,148 INFO warnings.warn("DWPose: Onnxruntime not found or doesn't come with acceleration providers, switch to OpenCV with CPU device. DWPose might run very slowly")
2024-02-26 20:09:02,865 INFO got prompt
2024-02-26 20:09:02,990 INFO model_type EPS
2024-02-26 20:09:02,990 INFO adm 0
2024-02-26 20:09:03,208 INFO Using pytorch attention in VAE
2024-02-26 20:09:03,209 INFO Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
2024-02-26 20:09:03,209 INFO Using pytorch attention in VAE
2024-02-26 20:09:03,614 INFO missing {'cond_stage_model.clip_l.text_projection', 'cond_stage_model.clip_l.logit_scale'}
2024-02-26 20:09:03,614 INFO left over keys: dict_keys(['cond_stage_model.clip_l.transformer.text_model.embeddings.position_ids'])
2024-02-26 20:09:03,708 INFO Requested to load SD1ClipModel
2024-02-26 20:09:03,708 INFO Loading 1 new model
2024-02-26 20:09:03,896 INFO D:\~diffuse\comfy-krita\ComfyUI\comfy\ldm\modules\attention.py:325: UserWarning: 1Torch was not compiled with flash attention. (Triggered internally at ..\aten\src\ATen\native\transformers\cuda\sdp_utils.cpp:263.)
2024-02-26 20:09:03,896 INFO out = torch.nn.functional.scaled_dot_product_attention(q, k, v, attn_mask=mask, dropout_p=0.0, is_causal=False)
2024-02-26 20:09:04,114 INFO [] []

**Comment by xunkar** [comment]
Update: I found out that this is happening when using "fill" but when using "generate (custom)" it works.

**Comment by Acly** [comment]
Is it specifically the "Blur" & "Border" modes? Probably #401 in that case - would be interesting what CPU you use, best we know is it affects AMD Ryzen 9 7000 series

**Comment by xunkar** [comment]
Sorry I'm new to Krita, I found the mode you're referring, it's set to Neutral. Everything except Generate (custom) works (fill, expand, etc.)

I'm using an Intel i7 11700K

**Comment by Acly** [comment]
Hm not 100% sure it's the same but it still sounds like it. I tried to add a workaround in [v1.15.0](https://github.com/Acly/krita-ai-diffusion/releases/tag/v1.15.0), please give it a try. Can't reproduce this myself, so it's hard to tell.

**Comment by xunkar** [comment]
Fixed in 1.15.0 👍
